atuin/src/command/client/history.rs

```diff
@@ -303,6 +307,12 @@ impl Cmd {
 
         print_list(&history, mode, format.as_deref());
 
+        if reverse {
+            let mut history = history;
```

Comment:
This can be achieved by changing the history to mutable and remove this temp variable here. 
Change the declaration of history above like this should be enough and more explicit
```
let mut history = match (session, cwd) {
```